skopt/learning/gaussian_process/gpr.py

```diff
@@ -197,3 +204,116 @@ def fit(self, X, y):
                     self.kernel_.set_params(
                         **{white_param: WhiteKernel(noise_level=0.0)})
         return self
+
+    def predict(self, X, return_std=False, return_cov=False,
+                return_mean_grad=False, return_std_grad=False):
+        """
+        In addition to the mean of the predictive distribution, also its
+        standard deviation (return_std=True) or covariance (return_cov=True),
+        the gradient of the mean and the gradient of the std can be optionally
+        provided.
```

Comment:
In the context of a GP, we should be more precise and specify with respect to what the gradients of mu and of std are computed (i.e. with respect to X, not with the hyper-params of the kernel)